GetHTML.py, 33, (<class 'requests.exceptions.ConnectionError'>, ConnectionError(MaxRetryError("HTTPSConnectionPool(host='water-sports-gadgets.bestselleroutlets.com%0a', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f3571bae208>: Failed to establish a new connection: [Errno -2] Name or service not known',))",),), <traceback object at 0x7f3571bcf148>)
